app/controllers/admin/assigned_routes_controller.rb

```diff
@@ -13,6 +13,8 @@ def create
       @form = AssignedRouteForm.new
 
       if @form.submit(assigned_route_params)
+        Mailer.assigned_route_mail(User.find(@form.user_id), @form.route)
```

Comment:
Add a `.deliver_now` for this mailer. Wasn't sending for me in development mode. Later we can switch this to `.deliver_later` once we have ActiveJob working correctly.

```ruby
Mailer.assigned_route_mail(User.find(@form.user_id), @form.route).deliver_now
```